main.py

```diff
@@ -20,15 +20,23 @@
 AUTH_FILE = sys.argv[1]
 
 
-def make_twitter_api():
+def make_api_keys():
```

Comment:
I would have this be two separate funtions: one here for twitter, one in `realtimetrains.py`...

Reply:
Done